src/js/views/patients/shared/list_views.js

```diff
@@ -51,6 +51,7 @@ const CountView = View.extend({
       maximumCount: MAXIMUM_LIST_COUNT,
       count: filteredCollection.length,
       isFlowList: !!this.getOption('isFlowList'),
+      totalInDb: this.getOption('totalInDb').toLocaleString(),
```

Comment:
Added `.toLocaleString()` here to add thousand comma separators to this number. Since these have the potential to be over `1,000`? What do you think?

Reply:
actually I think in the i18n `maximumListCount` you just do `{# totalInDb}`

I dunno if we've ever done it, but https://github.com/formatjs/handlebars-intl#example